process-controller/src/main/java/org/jboss/as/process/Main.java

```diff
@@ -221,7 +222,8 @@ public JBossThreadFactory run() {
         final InetSocketAddress boundAddress = processController.getServer().getBoundAddress();
 
         final List<String> initialCommand = new ArrayList<String>();
-        initialCommand.add(jvmName);
+        final JdkType jdkType = JdkType.createFromJavaExecutable(jvmName);
+        initialCommand.add(jdkType.getJavaExecutable());
```

Comment:
The changes to this file are not necessary.  There hasn't been a problem with the existing code and the new code here isn't doing anything relevant for JDK 11 support.

Eliminating this also mean JdkType can go in the same package as JvmOptionsBuilderFactory.

Reply:
Removing this fixes the test failures on Windows. https://github.com/wildfly/wildfly-core/pull/3478